client/coral-embed-stream/src/containers/Stream.js

```diff
@@ -112,7 +112,8 @@ class StreamContainer extends React.Component {
 
   componentDidMount() {
     if (this.props.previousTab) {
-      this.props.data.refetch();
+      this.props.data.refetch()
+        .then(() => this.props.setCommentCountCache(0));
```

Comment:
The `commentCountCache` basically should cache `asset.commentCount` from the last full query, which is then used to determine if new comments have arrived when the count polling returns a higher commentCount.